ui/src/app/store/subnet/types/base.ts

```diff
@@ -47,6 +87,13 @@ export type Subnet = Model & {
   vlan: VLAN["id"];
 };
 
+export type SubnetDetails = BaseSubnet & {
+  disabled_boot_architectures: Architecture[];
```

Comment:
It looks like this returned with the base subnet as well.

Reply:
oh you're right, oops